src/Domain/LeanCode.DomainModels/Ids/TypedIdConverter.cs

```diff
@@ -122,7 +122,9 @@ public override void WriteAsPropertyName(Utf8JsonWriter writer, TId value, JsonS
 
     private bool TryGetGuidCore(ref Utf8JsonReader reader, out Guid value)
     {
+#pragma warning disable RedundantAssignment
```

Comment:
Let's remove this `pragma warning disable`s that are ReSharper-only. They clutter the code more than anything.

Reply:
Okay, I've seen this has been changed to comments - they are much less clutter-y, they can stay.